src/libos/src/entry.rs

```diff
@@ -240,6 +252,36 @@ fn parse_arguments(
     Ok((path_buf, args, env_merged, host_stdio_fds))
 }
 
+fn parse_resolv_conf(resolv_conf_ptr: *const c_char) -> Result<()> {
+    // Read resolv.conf file from host
+    let resolv_conf_bytes = unsafe { CStr::from_ptr(resolv_conf_ptr).to_bytes() };
+    let resolv_conf_str =
+        str::from_utf8(resolv_conf_bytes).map_err(|_| errno!(ENOENT, "invalid symlink content"))?;
```

Comment:
`EINVAL` is more appropriate. And why is the error message `"invalid symlink content"`?

Reply:
Done